cmd/uhc/get/cmd.go

```diff
@@ -72,11 +72,36 @@ func init() {
 
 func run(cmd *cobra.Command, argv []string) {
 	// Check that there is exactly one command line parameter:
-	if len(argv) != 1 {
-		fmt.Fprintf(os.Stderr, "Expected exactly one argument\n")
+	if len(argv) < 1 || len(argv) > 2 {
+		cmd.Usage()
+		fmt.Fprintf(os.Stderr, cmd.Usage().Error())
```

Comment:
I think this should be `fmt.Fprintf(os.Stderr, "%v\n", cmd.Usage())`, otherwise `Fprintf` will interpret the error message a s format string, and that may result in unexpected results.

Reply:
done